src/Apps/Feature/FeatureApp.tsx

```diff
@@ -3,41 +3,52 @@ import { AppContainer } from "Apps/Components/AppContainer"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FeatureHeaderFragmentContainer as FeatureHeader } from "./Components/FeatureHeader"
 import { FeatureApp_feature } from "__generated__/FeatureApp_feature.graphql"
-import { Box, Sans, Spacer } from "@artsy/palette"
+import { Box, Join, Sans, Spacer } from "@artsy/palette"
+import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
+import { FeatureSetFragmentContainer as FeatureSet } from "./Components/FeatureSet"
 
 interface FeatureAppProps {
   feature: FeatureApp_feature
 }
 
 const FeatureApp: React.FC<FeatureAppProps> = ({ feature }) => {
   return (
-    <AppContainer maxWidth="100%">
+    <>
       <Box height="100vh">
         <FeatureHeader feature={feature} />
       </Box>
 
-      <Box maxWidth={["100%", 460]} mx="auto" my={3} px={3}>
-        <Sans size="4">
-          {/* Feature#description */}
-          Art-world spaces have paused, museums and galleries have closed their
-          doors, exhibitions have canceled, and auctions and fairs have been
-          postponed. Still, even in times of crisis, art can bring us closer and
-          keep us connected across time and place. And in times of crises, art
-          continues to move us and keeps going, and and keeps us connected.
-        </Sans>
+      <AppContainer>
+        <HorizontalPadding>
+          {(feature.description || feature.callOut) && (
+            <Box maxWidth={["100%", 460]} mx="auto" my={3} px={3}>
+              <Join separator={<Spacer my={3} />}>
+                {feature.description && (
+                  <Sans size="4">
+                    <Box
+                      dangerouslySetInnerHTML={{ __html: feature.description }}
```

Comment:
Is this our preferred way to render markdown?

Reply:
Would be nice if there was a more sanctioned way to do this but, we do want to render in the graph. @damassi is there a reason we don't expose this prop on Sans itself?

Reply:
Have never had the need (to my knowlege)

Reply:
What's weird is that it should just work there too — it's just a type error. There's a lot of machinery involved in setting the typography components up so maybe it's just getting lost.

One of the things to note here is that currently we're getting faux styles on any italics/bold that happens within Markdown rendered text. This is probably one of those things that you don't notice until you're aware of it but it. I'm gonna try to get some fixes into the font face declarations later in the week.